example-playbooks/replace_node/README.md

```diff
@@ -2,40 +2,50 @@
 
 This playbook will run the replace dead node procedure.
 
-### Steps:
+## Prerequisites
 
-1. Check if the node is a seed, if it is stop, seeds need to be demoted first.
-2. Apply the ansible-scylla-node role to the new node, reusing all the parameters from the existing cluster. Skip starting the scylla-server service
-3. Start the services
-4. Clean out the replace_address_first_boot record from scylla.yaml
-5. Run `nodetool repair -pr`
+* A scylla cluster with scylla-manager and scylla-monitoring installed.
+
+## Steps:
+
+1. Validate that the node being replaced is down
+2. Block the broadcast address of the replaced node via iptables in all the other nodes to prevent it from trying to join the cluster again
+3. Check if all the other nodes are up
+4. Install Scylla in the new node
+5. Set `replace_node_first_boot`/`replace_address_first_boot` in the `scylla.yaml` of the new node
+6. Start `scylla-server`
+7. Remove `replace_node_first_boot`/`replace_address_first_boot` from the `scylla.yaml` of the new node
+8. Start `scylla-manager-agent`
+9. Update Scylla-Monitoring
+10. If RBNO is disabled, repair the new node
+
+## Parameters:
+
+This playbook uses the node role to install and configure Scylla in the new node, so the same parameters
+used when the cluster was created should also be passed to the `replace_node.yml` playbook.
+In the `Usage` section of this README, we use `nodes_params.yml` and `monitor_params.yml` files to represent such parameters.
+
+Besides the vars from the node role, this playbook has the following mandatory parameters: `replaced_node`, `replaced_node_broadcast_address` and `new_node`.
+These and the other available parameters are listed and described in `vars/main.yml`.
 
 ## Usage:
 
-1. Reuse the existing set of parameters from the ansble-scylla-node run for the existing cluster. In this case I copy a previously created file called `nodes_params.yml` to `replace_params.yml`.
-2. Add the new node to the inventory:
+1. Add the new node to the inventory:
```

Comment:
This step in not optional. Hence you want to either mention it explicitly or reference it from the "Prerequisites" section.

Reply:
Done